### PR TITLE
Use Next Image on Home Page

### PR DESCRIPTION
**Pull request by AnishDe12020**
References #136 

## 📝 Description

The Chakra UI website had poor performance due to images not being served at the correct size and next-gen formats.

## ⛳️ Current behavior (updates)

Use of `chakra.img` and `Img` tags which don't do resizing of images or serve them in next-gen formats

## 🚀 New behavior

Use of `ChakraNextImage` tag for better performance

## 💣 Is this a breaking change (Yes/No):

No

## 📝 Additional Information

Change in lighthouse scores (left is chakra UI on prod and right is the preview deployment for this pr) - 
image

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -5,7 +5,14 @@ module.exports = withContentlayer()({
     optimizeFonts: true,
     modern: true,
   },
-  images: { domains: ['img.youtube.com'] },
+  images: {
+    domains: [
+      'img.youtube.com',
+      'assets-global.website-files.com',
+      'www.brikl.com',
```
Comment (lazarnikolov94) [inline next.config.js:12]: I would move these images locally, like we do for some of the sponsors. While we're here, can you do that?

**Review by lazarnikolov94 — CHANGES_REQUESTED** [review]

File: next.config.js (hunk 1 above)
Comment (AnishDe12020) [reply next.config.js:12]: Sure, I will do that.

**Comment by lazarnikolov94** [comment]
@AnishDe12020 looks good. Make sure you resolve the conflicts, then we can merge this.

**Comment by AnishDe12020** [comment]
> @AnishDe12020 looks good. Make sure you resolve the conflicts, then we can merge this.

Done, I have fixed the merge conflicts in https://github.com/chakra-ui/chakra-ui-docs/pull/137/commits/5bea83011b29011e68ffd079134d7176a3326c57. Do let me know if anything is wrong

**Comment by lazarnikolov94** [comment]
@allcontributors please add @AnishDe12020 for code.

**Pull request merged**